src/Components/Homepage/Carousel.css

```diff
@@ -0,0 +1,148 @@
+.test {
+  display: flex;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  position: relative;
+}
+html,
+body {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  font-size: 3vmin;
+}
+html {
+  background: transparent;
+  color: #fff;
+  /* overflow: hidden; */
+}
+body {
```

Comment:
after merge lets make sure changing style of the body element here doesn't altering styling of other components in the Home component